singer-connectors/target-snowflake/requirements.txt

```diff
@@ -1 +1 @@
-pipelinewise-target-snowflake==1.15.0
+-e git+https://github.com/transferwise/pipelinewise-target-snowflake.git@AP-1110#egg=pipelinewise-target-snowflake
```

Comment:
Will need to merge and release this https://github.com/transferwise/pipelinewise-target-snowflake/pull/254 before merging this PR